bitswap.go

```diff
@@ -508,6 +508,11 @@ func (bs *Bitswap) GetWantlist() []cid.Cid {
 	return bs.pm.CurrentWants()
```

Comment:
I'd suggest updating the comment as well to indicate it includes both want-blocks and want-haves